src/collective/collectionfilter/configure.zcml

```diff
@@ -1,79 +1,86 @@
 <configure
```

Comment:
Hi, I'm -1 on the reformatting.
The Plone and Zope style guides say:
```
Indentation of 2 characters to show nesting, 4 characters to list attributes on separate lines.
This distinction makes it easier to see the difference between attributes and nested elements.
```

https://docs.plone.org/develop/styleguide/python.html#indentation
https://zopetoolkit.readthedocs.io/en/latest/codingstyle/zcml-style.html

Reply:
@thet 
Thanks for the hint, good to know, i use vscode with "autoformat on save option" and the "XML Language Support by Red Hat" Plugin. I will refactor the zcml files with zpretty.